src/index.js

```diff
@@ -33,7 +33,7 @@ router.use(async (ctx, next) => {
 
 router.get("/:key", async ctx => {
   const { key } = ctx.params;
-
+  console.time(`reading key ${key}`);
```

Comment:
👍 

At some point this information is going to be probably too noisy.  Maybe we should integrate some metrics like `Prometheus` so we can access it through a endpoint like `/metrics` instead of terminal output.